### DISCUSSION TITLE
Is it possible, to Mark Text as Complete.

### DISCUSSION THREAD

**Discussion by YoussefEssalhi** [TomSchimansky/CustomTkinter#1873] [General]
Example : 

![Capture080405](https://github.com/TomSchimansky/CustomTkinter/assets/76540840/eb51795a-24b8-4e27-ba97-f8a8e35e0851)

**Comment by Akascape** [comment]
@YoussefEssalhi It is related to the font. You have to enable the overstrike parameter.
```python
CTkLabel(self, text="Hello World", font=CTkFont("", size=13, overstrike=1))
```

**Reply by YoussefEssalhi** [reply]
@Akascape Result :

![image](https://github.com/TomSchimansky/CustomTkinter/assets/76540840/8ed86193-2f70-4fb5-9c71-0029d16c6e21)

Is there any way to `center the line in text` ?

**Reply by Akascape** [reply]
@YoussefEssalhi Change the font family.
